.travis.yml

```diff
@@ -36,7 +36,9 @@ matrix:
   include:
     # Rails dev / 6 builds >= 2.4.4
     - rvm: jruby-head
```

Comment:
Don't we want to stick to one version in particular?

Reply:
There are recently issues installing JRuby on Travis that are unlikely to be repaired soon. I worked with @samphippen at RubyKaigi to get some version of JRuby running and this is the best we could get 😞